src/content/contributing/adding-layer-2/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Adding Layer 2's
+description: The policy we use when adding layer 2's to ethereum.org
+lang: en
+sidebar: true
+---
+
+# Adding layer 2's {#adding-layer-2}
+
+Anyone is free to suggest new layer 2's on ethereum.org.
+
+We currently list them on:
+
+- [ethereum.org/layer-2](/layer-2/)
+
+This page is a beginner/intermediate level page on layer 2's, and on this page a user is able to see some prominent layer 2 projects that they can easily get onboarded on.
+
+Because of this context, we need some specific information when you suggest a layer 2.
```

Comment:
I think we should ask for the same base level of information about any L2 project, regardless of where we put it (e.g. the beginner's page vs. the developer docs). Perhaps we ask for additional info for the beginner's page but we can start with a base requirement for any L2 listing.